### PR TITLE
CWC2018 RO16 Recap

### PR DESCRIPTION
**Pull request by JBHyperion**
Upload RO16 recap article with banner, twitch player and score images

Proposed release: 2018-06-22 at/after 1200 UTC+0

### CODE REVIEW

File: news/2018-06-22-cwc-2018-recap-round-of-16.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "osu!catch World Cup 2018: Round of 16 Recap"
+date: 2018-06-22 12:00:00 +0000
+---
+
+Two weeks of frantic fruit-catching in the 2018 osu!catch World Cup completed, and several more still to look forward to - but who triumphed at the top of the tree, and who fell off the platter of progress in the Round of 16? Read on to find out more!
```
Comment (Joehuu) [inline news/2018-06-22-cwc-2018-recap-round-of-16.md]: `2018 osu!catch World Cup` -> `osu!catch World Cup 2018`

File: news/2018-06-22-cwc-2018-recap-round-of-16.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+![](/wiki/shared/news/banners/CWC_2018_banner.jpg)
+
+Hello once again one and all, [JBHyperion](https://osu.ppy.sh/users/4879508) here to deliver your weekly round-up on all the action from this year’s edition of the osu!catch World Cup. Following on from one of the most incredible Group Stages ever seen in an osu! World Cup, last weekend’s Round of 16 saw our remaining teams each jostling for position in their march to becoming this year’s *osu!catch Champions*.
```
Comment (Joehuu) [inline news/2018-06-22-cwc-2018-recap-round-of-16.md]: `’` -> `'`
The apostrophe is not the correct character unicode. Correct the other instances.

File: news/2018-06-22-cwc-2018-recap-round-of-16.md (hunk 1 above)
Comment (JBHyperion) [reply news/2018-06-22-cwc-2018-recap-round-of-16.md]: Reversed, might used "2018 edition of..." in future

File: news/2018-06-22-cwc-2018-recap-round-of-16.md (hunk 2 above)
Comment (JBHyperion) [reply news/2018-06-22-cwc-2018-recap-round-of-16.md]: Imagine still having to fix these in 2018... Sigh, '-ified al

File: news/2018-06-22-cwc-2018-recap-round-of-16.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+layout: post
+title: "osu!catch World Cup 2018: Round of 16 Recap"
+date: 2018-06-22 12:00:00 +0000
+---
+
+Two weeks of frantic fruit-catching in the osu!catch World Cup 2018 completed, and several more still to look forward to - but who triumphed at the top of the tree, and who fell off the platter of progress in the Round of 16? Read on to find out more!
+
+![](/wiki/shared/news/banners/CWC_2018_banner.jpg)
+
+Hello once again one and all, [JBHyperion](https://osu.ppy.sh/users/4879508) here to deliver your weekly round-up on all the action from this year's edition of the osu!catch World Cup. Following on from one of the most incredible Group Stages ever seen in an osu! World Cup, last weekend's Round of 16 saw our remaining teams each jostling for position in their march to becoming this year's *osu!catch Champions*.
```
Comment (osuShiro) [inline news/2018-06-22-cwc-2018-recap-round-of-16.md]: "our remaining teams each jostling for position in their march" is sort of poorly worded - change it for "for a position in the march"

File: news/2018-06-22-cwc-2018-recap-round-of-16.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+A further eight matches were shown live on stream this past Sunday, each delivering further energy and excitement, shocks and swerves, as the tournament bracket continues to take shape.
```
Comment (osuShiro) [inline news/2018-06-22-cwc-2018-recap-round-of-16.md]: "further" is repeated

File: news/2018-06-22-cwc-2018-recap-round-of-16.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+A further eight matches were shown live on stream this past Sunday, each delivering further energy and excitement, shocks and swerves, as the tournament bracket continues to take shape. 
+
+Wanting to inject a little more flair and personality into things, the first match was preceded by an osu! Coffee Hour “pre-show” where fellow commentator [Dohland](https://osu.ppy.sh/u/5220511) and myself ran down a recap of the best moments from the previous week's Group Stages, as well as most anticipated moments and predictions for each of the Round of 16 matches.
+
+If you missed any of the action and analysis, or are just eager to relive it a second time, VODs for all eight matches and pre-show are available on the [osu!live Twitch channel](https://www.twitch.tv/osulive).
+
+<iframe src="https://player.twitch.tv/?autoplay=false&video=v274192014" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="100%"></iframe>\
```
Comment (osuShiro) [inline news/2018-06-22-cwc-2018-recap-round-of-16.md]: is this appended backstroke intended ?

File: news/2018-06-22-cwc-2018-recap-round-of-16.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+Another team who were left wondering “what if?” after their match were The Netherlands, whose strategy unraveled spectacularly after only a single game. Attempting to sneak an opening victory on the uncomfortable NoMod pick, [sana - Packet Hero](https://osu.ppy.sh/beatmapsets/694901#fruits/1469957), the Dutch team succumbed to a 15,500 point loss as the Germans held their nerve in a frantic back-and-forth exchange which had everyone out of their seats.
+
+![](/wiki/shared/news/2018-06-22-cwc-2018-recap-round-of-16/GERvNL_game1_720w.jpg)
+
+From this point The Netherlands confidence was shot, and, buoyed by their opening break point victory, Germany reeled off four further game wins in succession to complete the 5-0 sweep. This was some measure of revenge for Germany, who had been soundly beaten by the Dutch in last year’s Semifinals, and they played with neither fear nor mercy in a frightening display of consistency that their future opponents have surely taken notice of.
```
Comment (osuShiro) [inline news/2018-06-22-cwc-2018-recap-round-of-16.md]: uncapitalize "The" in the first sentence

File: news/2018-06-22-cwc-2018-recap-round-of-16.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+The Round of 16 concluded with a clinical United States securing a comfortable 5-0 victory over Sweden. Sweden threw quite literally everything in their playbook at this contest, but it merely bounced right off the American armour. Game after game, the Swedish challenge was checked by the Americans, who played with unshakeable consistency throughout and look incredibly strong going forward.
+
+## Upcoming Stage - Quarter Finals
+
+Whilst no further eliminations will occur until next week, our eight Losers' Bracket teams must be feeling the nerves knowing every match is now a fight for survival. On the top side of the draw, the Netherlands look to recover against an inconsistent Japan, whilst Finland seek to continue their impressive run against a struggling Mexico. On the bottom side, Poland and Sweden look to bounce back from major losses, whilst the similarly over-performing France and Norway clash in what could prove to be a gutsy battle of attrition.
```
Comment (osuShiro) [inline news/2018-06-22-cwc-2018-recap-round-of-16.md]: "whilst" repeated three times :x

Comment (osuShiro) [reply news/2018-06-22-cwc-2018-recap-round-of-16.md]: "look to" is also repeated

File: news/2018-06-22-cwc-2018-recap-round-of-16.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+If any of the remaining Winners' Bracket teams see this as a comfort net, they will no doubt receive a harsh reality check come the weekend, as the skill level and tournament experience on display is honestly staggering to behold. Germany look to prey upon an inconsistent South Korea in the top half of the draw, joined by a strong Chilean outfit and the ever-dangerous Taiwan.
```
Comment (osuShiro) [inline news/2018-06-22-cwc-2018-recap-round-of-16.md]: "look to" is also repeated here

File: news/2018-06-22-cwc-2018-recap-round-of-16.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+The bottom half once again resembles a veritable warzone, with four of 2017's top six teams staring each other down for a Semi Final spot. Indonesia first cross blades with Hong Kong in a match oozing with star power and limitless potential, but the match on everybody's must watch list will surely be China locking horns with the United States. This mega match could realistically be a preview of this year's Grand Final, given the incredibly dominant showings by these two juggernauts of the fruit catching world thus far in the tournament, so is absolutely not to be missed!
+
+## Summary
+
+Whether you're a long time fruit-catcher or had never checked out osu!catch before this article, everyone is welcome (and encouraged!) to join us for the Quarter Finals this coming weekend. Follow [@osugame on Twitter](https://twitter.com/osugame) to keep in touch, stay updated, and know when we go live, support your favourite knockout stage team by purchasing a profile banner from the [osu!store](https://osu.ppy.sh/store/listing), and of course, head on over to the [osu!live Twitch channel](https://www.twitch.tv/osulive) to witness all the action as it unfolds.
```
Comment (osuShiro) [inline news/2018-06-22-cwc-2018-recap-round-of-16.md]: there needs to be a stylistic break between the "follow osulive" stuff and the "support your team" one

Comment (osuShiro) [reply news/2018-06-22-cwc-2018-recap-round-of-16.md]: semicolons would do, just don't forget one before "and of course"

File: news/2018-06-22-cwc-2018-recap-round-of-16.md (hunk 3 above)
Comment (JBHyperion) [reply news/2018-06-22-cwc-2018-recap-round-of-16.md]: Sounds okay to me, but I agree your wording is slightly better so sure

File: news/2018-06-22-cwc-2018-recap-round-of-16.md (hunk 4 above)
Comment (JBHyperion) [reply news/2018-06-22-cwc-2018-recap-round-of-16.md]: Changed to "Eight more"

File: news/2018-06-22-cwc-2018-recap-round-of-16.md (hunk 5 above)
Comment (JBHyperion) [reply news/2018-06-22-cwc-2018-recap-round-of-16.md]: Using it to force a new line, my previous github markdown of </br> to do this didn't seem to work as intended

File: news/2018-06-22-cwc-2018-recap-round-of-16.md (hunk 6 above)
Comment (JBHyperion) [reply news/2018-06-22-cwc-2018-recap-round-of-16.md]: Fixed and also added comma

File: news/2018-06-22-cwc-2018-recap-round-of-16.md (hunk 7 above)
Comment (JBHyperion) [reply news/2018-06-22-cwc-2018-recap-round-of-16.md]: Creativity skill zero

File: news/2018-06-22-cwc-2018-recap-round-of-16.md (hunk 8 above)
Comment (JBHyperion) [reply news/2018-06-22-cwc-2018-recap-round-of-16.md]: Fixed

File: news/2018-06-22-cwc-2018-recap-round-of-16.md (hunk 9 above)
Comment (JBHyperion) [reply news/2018-06-22-cwc-2018-recap-round-of-16.md]: Why was this sentence so god damn long yuck, split these up into separate sentences

**Pull request merged**